app/code/Magento/Sales/view/frontend/templates/order/print/invoice.phtml

```diff
@@ -32,11 +32,10 @@
             </tr>
             </thead>
             <?php $_items = $_invoice->getItemsCollection(); ?>
-            <?php $_count = $_items->count(); ?>
             <?php foreach ($_items as $_item): ?>
-            <?php if ($_item->getOrderItem()->getParentItem()) {
-    continue;
-} ?>
+                <?php if ($_item->getOrderItem()->getParentItem()): ?>
+                    continue;
```

Comment:
The same issue it there.

Reply:
@ihor-sviziev Thank you so much for your review. Sorry, it was my fail :(

Reply:
@Coderimus no problem. I just created new PR that fixing this issue